Mathlib/Algebra/Order/Star/Basic.lean

```diff
@@ -44,6 +44,19 @@ universe u
 
 variable {R : Type u}
 
+/- This lemma is placed in this file instead of `Algebra.Star.SelfAdjoint` for two reasons:
+1. `AddSubmonoid.closure` is not available in that file.
+2. This is used below to show `StarOrderedRing Rᵐᵒᵖ`.
```

Comment:
```suggestion
/- This lemma is placed in this file instead of `Algebra.Star.SelfAdjoint` because
`AddSubmonoid.closure` is not available in that file.

```